presto-spi/src/main/java/com/facebook/presto/spi/CoordinatorPlugin.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi;
+
+import com.facebook.presto.spi.function.FunctionNamespaceManagerFactory;
+
+import static java.util.Collections.emptyList;
+
+public interface CoordinatorPlugin
```

Comment:
Can you add some Javadoc to describe this plugin?  ie. what it is used for, what differentriates it from Plugin?